packages/rulesets/src/asyncapi/index.ts

```diff
@@ -142,6 +143,31 @@ export default {
         function: truthy,
       },
     },
+    'asyncapi-message-examples': {
+      description: 'Message...',
```

Comment:
should we add some? :D

Reply:
Yeah 😅  Updated! Thanks for pointing that out!